src/tools/linkchecker/main.rs

```diff
@@ -387,7 +387,7 @@ fn maybe_redirect(source: &str) -> Option<String> {
     const REDIRECT: &str = "<p>Redirecting to <a href=";
 
     let mut lines = source.lines();
-    let redirect_line = lines.nth(6)?;
+    let redirect_line = lines.nth(7)?;
```

Comment:
For information: this change is needed because of my change in the redirection file which added a title to the page.